src/web-auth/index.js

```diff
@@ -929,4 +930,20 @@ WebAuth.prototype.passwordlessVerify = function(options, cb) {
   });
 };
 
+/**
+ *
+ * Renders the captcha challenge in the provided element.
+ * This function can only be used in the context of a Classic Universal Login Page.
```

Comment:
What happens when it is _not_ used in the context of the Classic ULP?

Reply:
If the captcha is not solved in the context of a classic ULP transaction that was flagged as suspicious  the request will fail with an error.